Document/0x05a-Platform-Overview.md

```diff
@@ -48,60 +105,103 @@ For example, Android 7.0 (API level 24) defines the following system users:
     ...
 ```
 
-<br/>
-<br/>
+##### SELinux
 
-#### Android Device Encryption
+Security-Enhanced Linux (SELinux) uses a Mandatory Access Control (MAC) system to further lock down which processes should have access to which resources. Each resource is given a label in the form of `user:role:type:mls_level` which defines which users are able to execute which types of actions on it. For example, one process may only be able to read a file, while another process may be able to edit or delete the file. This way, by working on a least-privilege principle, vulnerable processes are more difficult to exploit via privilege escalation or lateral movement.
 
-Android supports device encryption from Android 2.3.4 (API level 10) and it has undergone some big changes since then. Google imposed that all devices running Android 6.0 (API level 23) or higher had to support storage encryption. Although some low-end devices were exempt because it would significantly impact performance. In the following sections you can find information about device encryption and its algorithms.
+Further information is available on the [Android Security website](https://source.android.com/security/selinux "Security-Enhanced Linux in Android").
 
-##### Full-Disk Encryption
+##### Permissions
 
-Android 5.0 (API level 21) and above support full-disk encryption. This encryption uses a single key protected by the users' device password to encrypt and decrypt the userdata partition. This kind of encryption is now considered deprecated and file-based encryption should be used whenever possible. Full-disk encryption has drawbacks, such as not being able to receive calls or not having operative alarms after a reboot if the user does not enter the password to unlock.
+Android implements an extensive permissions system that is used as an access control mechanism. They are used to control access to sensitive user data and device resources. Prior to Android 6.0 (API level 23), all permissions an app requested were granted at installation. From API level 23 onwards, the user must approve some permissions requests during runtime.

```

Comment:
> They are

This refers to something plural, but the previous sentence only has singular nouns.